### ISSUE TITLE
[YSQL] separate out keyspaces / databases in ycql, ysql and yedis namespaces

### ISSUE THREAD

**Issue by rkarthik007** [YugaByte/yugabyte-db#255]
Hitting the following error:
```
SQL error (yb/yql/pgsql/ptree/pg_process_context.cc:182): Database Already Exists. Already present (yb/common/wire_protocol.cc:153): Keyspace myapp already exists: b84fa990220e46988491cc1f745d4d3b
```

**Comment by ravimurthy** [comment]
Updated notes relevant to this bug.

There's a weird behavior with postgres databases showing up as cassandra keyspaces. I am also able to create CQL table within this database, and describe the keyspace, but "use" operation fails.  The cleanest solution would be to filter away pg databases in cql.
===
"sample" is a postgres database.

cqlsh> describe keyspaces;
template1  template0  postgres  system_schema  system_auth  system  sample

cqlsh> create table sample.foo (a text, b text, primary key (a));
-- WORKS

cqlsh> use sample
  ... ;
InvalidRequest: Error from server: code=2200 [Invalid query] message="SQL error: Keyspace Not Found. Not found: Cannot use unknown keyspace
use sample
^^^
;
(error -308)"
cqlsh> describe keyspace sample;

CREATE KEYSPACE sample WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '3'}  AND durable_writes = true;

CREATE TABLE sample.foo (
   a text PRIMARY KEY,
   b text
) WITH default_time_to_live = 0
   AND transactions = {'enabled': 'false'};

**Comment by robertpang** [comment]
Fixed in commit [0037132](https://github.com/YugaByte/yugabyte-db/commit/0037132f30f67fd2e0813920c609e019f8657293).
